### ISSUE TITLE
Divider, Input not a contructor for React Elements

### ISSUE THREAD

**Issue by tomitrescak** [Semantic-Org/Semantic-UI-React#1818]
### Steps

use `<Input /> or <Divider /> or <Loader />` in typescript code

### Expected Result

No 

### Actual Result

Following error is reported:

```
[ts]
JSX element type 'Divider' is not a constructor function for JSX elements.
  Types of property 'setState' are incompatible.
    Type '{ <K extends never>(f: (prevState: void, props: DividerProps) => Pick<void, K>, callback?: () => ...' is not assignable to type '{ <K extends never>(f: (prevState: {}, props: any) => Pick<{}, K>, callback?: () => any): void; <...'.
      Types of parameters 'f' and 'f' are incompatible.
        Types of parameters 'prevState' and 'prevState' are incompatible.
          Type 'void' is not assignable to type '{}'.
```
### Version
0.68.5

### Testcase
[Fork, update, and replace this pen to show the bug]:

;( Can't do ...

**Comment by tomitrescak** [comment]
Not sure how you guys generate the .d.ts file these days but following seems to do the trick:

Change from

```ts
export class Loader extends React.Component<LoaderProps, void> { }
```

to

```ts
var Divider: React.ComponentClass<DividerProps, void>;
```

**Comment by layershifter** [comment]
I'm not sure that you're using `0.68.5`, `0.68.0` [has](https://github.com/Semantic-Org/Semantic-UI-React/blob/v0.68.0/src/elements/Divider/Divider.d.ts) already correct export:

```ts
declare const Divider: React.StatelessComponent<DividerProps>;
```

`Loader`, BTW, too.

**Comment by tomitrescak** [comment]
@layershifter ... yup, embarrassing ... got older .d.ts pulled out on the side clashing.

**Comment by layershifter** [comment]
However, thanks for the attention, I shipped #1820 that will fix incorrect definition of `Input` - it's a class component, not stateless.

I also pulled a [boilerplate](https://github.com/rokoroku/react-redux-typescript-boilerplate) and tested with latest, all works fine.
